This file is part of MXE. See LICENSE.md for licensing information.

From 9aad92fdf482f1a38360559586db9ee2df5d9fc4 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Sun, 29 Jan 2017 20:27:29 +0100
Subject: [PATCH] build fix for MXE

Needed for the shared build. In the static build, gdi32 is probably brought in as a transitive dependency.

Change-Id: Ic0b1ab49cfb04559823c7446bda197b34ea15f54

diff --git a/src/winextras/winextras.pro b/src/winextras/winextras.pro
index 5d93c6b..4c423b0 100644
--- a/src/winextras/winextras.pro
+++ b/src/winextras/winextras.pro
@@ -45,7 +45,7 @@ HEADERS += \
 
 QMAKE_DOCS = $$PWD/doc/qtwinextras.qdocconf
 
-LIBS_PRIVATE += -lole32 -lshlwapi -lshell32 -ldwmapi
+LIBS_PRIVATE += -lole32 -lshlwapi -lshell32 -ldwmapi -lgdi32
 win32:!qtHaveModule(opengl)|qtConfig(dynamicgl):LIBS_PRIVATE += -lgdi32
 
 OTHER_FILES += \
-- 
2.9.3

